lib/src/validation-server/server.spec.ts

```diff
@@ -0,0 +1,26 @@
+import request from "supertest";
+import { parse } from "../neu/parser";
+import { runValidationServer } from "./server";
+
+const CONTRACT_PATH = "./lib/src/__examples__/contract.ts";
+
+describe("Server", () => {
+  const mockLogger = {
+    log: (message: string) => message
+  };
+
+  const contract = parse(CONTRACT_PATH);
+
+  describe("Run", () => {
+    it("/health and return 200", done => {
+      const { app } = runValidationServer(5907, contract, mockLogger);
```

Comment:
It's easy to think that the number `5907` has a specific meaning for the test. Maybe put it in a constant called `DUMMY_PORT`?